src/main/java/org/symphonyoss/symphony/messageml/bi/BiFields.java

```diff
@@ -2,84 +2,89 @@
+  TABLE_CELL_ROW_SPAN("TableCellsRowSpan", BiEventType.MESSAGEML_MESSAGE_SENT),
+  TABLE_HEADER("TableHeaders", BiEventType.MESSAGEML_MESSAGE_SENT),
+  TABLE_FOOTER("TableFooters", BiEventType.MESSAGEML_MESSAGE_SENT),
+  EMOJIS("Emojis", BiEventType.MESSAGEML_MESSAGE_SENT),
+  MENTIONS("Mentions", BiEventType.MESSAGEML_MESSAGE_SENT),
+  HASHTAGS("Hashtags", BiEventType.MESSAGEML_MESSAGE_SENT),
+  CASHTAGS("Cashtags", BiEventType.MESSAGEML_MESSAGE_SENT),
+  BULLET_LIST("Lists", BiEventType.MESSAGEML_MESSAGE_SENT),
+  CARD("Cards", BiEventType.MESSAGEML_MESSAGE_SENT),
+  EXPANDABLE_CARDS("ExpandableCards", BiEventType.MESSAGEML_MESSAGE_SENT),
+  EXPANDABLE_CARDS_COLLAPSED("ExpandableCardsCollapsed", BiEventType.MESSAGEML_MESSAGE_SENT),
+  EXPANDABLE_CARDS_CROPPED("ExpandableCardsCropped", BiEventType.MESSAGEML_MESSAGE_SENT),
+  EXPANDABLE_CARDS_EXPANDED("ExpandableCardsExpanded", BiEventType.MESSAGEML_MESSAGE_SENT),
+  HEADER("Headers", BiEventType.MESSAGEML_MESSAGE_SENT),
+  IMAGE("Images", BiEventType.MESSAGEML_MESSAGE_SENT),
+  IMAGE_DATA("ImagesData", BiEventType.MESSAGEML_MESSAGE_SENT),
+  IMAGE_URL("ImagesUrl", BiEventType.MESSAGEML_MESSAGE_SENT),
+  LINE_BREAK("LineBreaks", BiEventType.MESSAGEML_MESSAGE_SENT),
+  INPUT_STEP("Input_Step", BiEventType.MESSAGEML_ELEMENT_SENT),
+  TITLE("Title", BiEventType.MESSAGEML_ELEMENT_SENT),
+  DEFAULT("Default", BiEventType.MESSAGEML_ELEMENT_SENT),
+  PLACEHOLDER("Placeholder", BiEventType.MESSAGEML_ELEMENT_SENT),
+  STYLE_COLOR("StyleColor", BiEventType.MESSAGEML_ELEMENT_SENT),
+  STYLES_CUSTOM("StylesCustom", BiEventType.MESSAGEML_MESSAGE_SENT),
+  STYLES_CLASS_TEMPO("StylesClassTempo", BiEventType.MESSAGEML_MESSAGE_SENT),
+  STYLES_CLASS_OTHER("StylesClassOther", BiEventType.MESSAGEML_MESSAGE_SENT),
+  TYPE("Type", BiEventType.MESSAGEML_ELEMENT_SENT),
+  TYPE_MASKED_TRUE("Masked", BiEventType.MESSAGEML_ELEMENT_SENT),
+  TYPE_MASKED_FALSE("Normal", BiEventType.MESSAGEML_ELEMENT_SENT),
+  ENTITY_TYPE("EntityType", BiEventType.MESSAGEML_ELEMENT_SENT),
+  LABEL("Label", BiEventType.MESSAGEML_ELEMENT_SENT),
+  OPTIONS_COUNT("OptionsCount", BiEventType.MESSAGEML_ELEMENT_SENT),
+  VALIDATION("Validation", BiEventType.MESSAGEML_ELEMENT_SENT),
+  VALIDATION_MAX("Validation_Max", BiEventType.MESSAGEML_ELEMENT_SENT),
+  VALIDATION_MIN("Validation_Min", BiEventType.MESSAGEML_ELEMENT_SENT),
+  VALIDATION_PATTERN("Validation_Pattern", BiEventType.MESSAGEML_ELEMENT_SENT),
+  VALIDATION_OPTIONS("Validation_Options", BiEventType.MESSAGEML_ELEMENT_SENT),
+  HIGHLIGHTED_OPTIONS("HighlightedOptions", BiEventType.MESSAGEML_ELEMENT_SENT),
+  VALIDATION_STRICT("Validation_Strict", BiEventType.MESSAGEML_ELEMENT_SENT),
+  REQUIRED("Required", BiEventType.MESSAGEML_ELEMENT_SENT),
+  MESSAGE_LENGTH("MessageLength", BiEventType.MESSAGEML_MESSAGE_SENT),
+  ENTITY_JSON_SIZE("EntitiesJSONSize", BiEventType.MESSAGEML_MESSAGE_SENT),
+  FREEMARKER("UseFreeMarker", BiEventType.MESSAGEML_ELEMENT_SENT),
+  COUNT("Count", null);
```

Comment:
is there a way we can avoid null? (by having a dedicated BiEvenType maybe, like None?